language/transaction-builder/generator/tests/generation.rs

```diff
@@ -222,7 +222,6 @@ fn test_that_cpp_code_compiles_and_demo_runs() {
 }
 
 #[test]
-#[ignore]
```

Comment:
We shouldn't unignore this.

Reply:
Ugh. Good catch. I removed it for local testing. Thanks.